packages/dev-utils/lib/manifest-parser/impl.ts

```diff
@@ -25,6 +25,12 @@ function convertToFirefoxCompatibleManifest(manifest: Manifest) {
   manifestCopy.content_security_policy = {
     extension_pages: "script-src 'self'; object-src 'self'",
   };
+  manifestCopy.browser_specific_settings = {
+    gecko: {
+      id: 'example@example.com',
+      strict_min_version: '109.0',
```

Comment:
manifest v3 support enabled from version FF 109.0
id, maybe should come from env, or some other field of manifest.json.
At the end of the day, it needs valid email here to make it work